### PR TITLE
[PHP] Fix string interpolations

### PR DESCRIPTION
**Pull request by deathaxe**

### CODE REVIEW

File: PHP/PHP.sublime-syntax [hunk 1]
```diff
@@ -21,42 +21,62 @@ first_line_match: '^(#!.*[^-]php[0-9]?|<\?php)\b'
...
-        - meta_content_scope: source.php.embedded.html
-        - include: php-end-tag-pop
-        - include: scope:source.php
-          apply_prototype: true
-
-  php-end-tag-pop:
-    - match: (\?>)(\s*\n)?
-      captures:
-        1: punctuation.section.embedded.end.php
-        2: meta.html-newline-after-php.php
```
Comment (FichteFoll) [inline PHP/PHP.sublime-syntax:58]: I missed that this match was removed, too, as I thought it was only done for JS and CSS. The scope is needed for indentation rules and causes the indentation tests to fail.

I can reintroduce it tomorrow if you don't beat me to it. :wink:

Comment (deathaxe) [reply PHP/PHP.sublime-syntax:58]: Wasn't aware about it being used for indentation. Is that something to consider for the other syntaxes, too, then?

Comment (FichteFoll) [reply PHP/PHP.sublime-syntax:58]: I'm not sure. I barely spent time on this, but I'm sure just asking @keith-hall would be quicker. :wink:

Comment (keith-hall) [reply PHP/PHP.sublime-syntax:58]: > Wasn't aware about it being used for indentation. Is that something to consider for the other syntaxes, too, then?

Yep, though maybe we should tweak the scope a bit to standardize it in a more general form. It's also a good time to add indentation tests for other syntaxes :)

Comment (deathaxe) [reply PHP/PHP.sublime-syntax:58]: At least, I've learned why I missed those tests (https://github.com/sublimehq/sublime_text/issues/4113).

It appears PHP's sophisticated indentation rules may be used for other syntaxes as well. Ruby on Rails seems to be a candidate, maybe JSP as well.

Yes, indentation tests are useful, but I often failed to implement them due to ST's limitations, which prevent proper indentation in many situations anyway.

Comment (FichteFoll) [reply PHP/PHP.sublime-syntax:58]: Addressed in c1b69b45a69ee0734a522be4f7232da69368267f and 71c32a3f2a064c5dd392b4af9871bbef93f5a7b8.

**Pull request merged**